frontend/src/api/actions/init.js

```diff
@@ -0,0 +1,150 @@
+import {
+    setActiveSession,
+    // setActiveUserRole,
+    fetchActiveUser,
+    fetchSessions,
+    fetchApplicants,
+    fetchApplications,
+    fetchAssignments,
+    fetchContractTemplates,
+    fetchInstructors,
+    fetchPositions
+} from "./";
+import { setGlobals, globalsSelector } from "../../views/globals/actions";
+
+const toggleMockApi = (getState, newMockAPIValue) => {
```

Comment:
`newMockAPIValue` -> `enableMockAPI`

Reply:
We shouldn't use arrow functions where they aren't needed/don't make the code more concise. Do `function toggleMockApi()` instead